src/lib/components/wizard.component.ts

```diff
@@ -108,7 +112,13 @@ export class WizardComponent implements OnChanges, AfterContentInit {
    *
    * @param model The model for this wizard component
    */
-  constructor(public model: WizardState) {
+  constructor(
+    public model: WizardState,
+    // Using @Optional() in order not to break applications which import ArchwizardModule without calling forRoot().
+    @Optional() @Inject(NAVIGATION_MODE_FACTORY) private navigationModeFactory: NavigationModeFactory) {
```

Comment:
Add `@param navigationModeFactory ...` to the comments.

Reply:
Sorry, missed that. Added in 7d9804a.